.github/workflows/ci-go.yml

```diff
@@ -0,0 +1,46 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - '.github/workflows/ci-go.yml'
+      - '**.go'
+      - 'go.*'
+      - 'testdata/**'
+  pull_request:
+    paths:
+      - '.github/workflows/ci-go.yml'
+      - '**.go'
+      - 'go.*'
+      - 'testdata/**'
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up Go
+        uses: actions/setup-go@v4
+        with:
+          go-version-file: 'go.mod'
+          cache-dependency-path: 'go.sum'
+      - uses: davidB/rust-cargo-make@v1
```

Comment:
https://github.com/davidB/rust-cargo-make/pull/178

Why not yet merged...? 🤔

Reply:
https://github.com/kachick/dotfiles/pull/185/commits/dc132dc04cf604b39d93018f735a171e83257c1f => https://github.com/kachick/dotfiles/pull/185/commits/d2c9cae0eb347b4fa698a16167fa8659a461863e